source/lib-intro.tex

```diff
@@ -1614,15 +1614,15 @@
 \begin{codeblock}
 #include <utility>
```

Comment:
Drive-by: since this is an unusually complete example trying to illustrate what user code would look like, perhaps we should `#include <cassert>` here?